src/WebJobs.Script.WebHost/Controllers/AdminController.cs

```diff
@@ -33,12 +35,14 @@ public class AdminController : ApiController
         private readonly WebScriptHostManager _scriptHostManager;
         private readonly WebHostSettings _webHostSettings;
         private readonly TraceWriter _traceWriter;
+        private readonly ILogger _logger;
 
-        public AdminController(WebScriptHostManager scriptHostManager, WebHostSettings webHostSettings, TraceWriter traceWriter)
+        public AdminController(WebScriptHostManager scriptHostManager, WebHostSettings webHostSettings, TraceWriter traceWriter, ILoggerFactory loggerFactory)
         {
             _scriptHostManager = scriptHostManager;
             _webHostSettings = webHostSettings;
             _traceWriter = traceWriter.WithSource($"{ScriptConstants.TraceSourceHostAdmin}.Api");
+            _logger = loggerFactory?.CreateLogger("Host.Admin.Api");
```

Comment:
I do have a potential solution -- we've punted it until after this initial commit is in as it'll require a bunch of refactoring. Dispatching likely won't work very easily due to how ILoggers and TraceWriters are constructed (you have many ILoggers; few TraceWriters). But I'll experiment.

The string literal is temporary; I'll move it to a constant shortly. Looking for feedback on the right names, but that's not super-important for this initial commit as Categories, etc, are things that we can tweak during Preview. I can't use the existing ones because they don't follow the category 'namespacing' that I'm using for Loggers (i.e. everything from the host is 'Host.{category1}.{category2').